local-cli/generator-macos/templates/macos/Podfile

```diff
@@ -1,112 +1,37 @@
-  pod 'React-RCTAnimation', :path => '../node_modules/react-native-macos/Libraries/NativeAnimation'
-  pod 'React-RCTBlob', :path => '../node_modules/react-native-macos/Libraries/Blob'
-  pod 'React-RCTImage', :path => '../node_modules/react-native-macos/Libraries/Image'
-  pod 'React-RCTLinking', :path => '../node_modules/react-native-macos/Libraries/LinkingIOS'
-  pod 'React-RCTNetwork', :path => '../node_modules/react-native-macos/Libraries/Network'
-  pod 'React-RCTSettings', :path => '../node_modules/react-native-macos/Libraries/Settings'
-  pod 'React-RCTText', :path => '../node_modules/react-native-macos/Libraries/Text'
-  pod 'React-RCTVibration', :path => '../node_modules/react-native-macos/Libraries/Vibration'
-  pod 'React-Core/RCTWebSocket', :path => '../node_modules/react-native-macos/'
-
-  pod 'React-cxxreact', :path => '../node_modules/react-native-macos/ReactCommon/cxxreact'
-  pod 'React-jsi', :path => '../node_modules/react-native-macos/ReactCommon/jsi'
-  pod 'React-jsiexecutor', :path => '../node_modules/react-native-macos/ReactCommon/jsiexecutor'
-  pod 'React-jsinspector', :path => '../node_modules/react-native-macos/ReactCommon/jsinspector'
-  pod 'React-callinvoker', :path => '../node_modules/react-native-macos/ReactCommon/callinvoker'
-  pod 'ReactCommon/turbomodule/core', :path => '../node_modules/react-native-macos/ReactCommon'
-  pod 'Yoga', :path => '../node_modules/react-native-macos/ReactCommon/yoga', :modular_headers => true
-
-  pod 'DoubleConversion', :podspec => '../node_modules/react-native-macos/third-party-podspecs/DoubleConversion.podspec'
-  pod 'glog', :podspec => '../node_modules/react-native-macos/third-party-podspecs/glog.podspec'
-  pod 'RCT-Folly', :podspec => '../node_modules/react-native-macos/third-party-podspecs/RCT-Folly.podspec'
-  pod 'boost-for-react-native', :podspec => '../node_modules/react-native-macos/third-party-podspecs/boost-for-react-native.podspec'
-
-  target 'HelloWorld-macOS' do
-    platform :macos, '10.13'
-    use_native_modules!
 
-    # Enables Hermes
-    #
-    # Be sure to first install the `hermes-engine-darwin` npm package, e.g.:
-    #
-    #   $ yarn add 'hermes-engine-darwin@^0.4.3'
-    #
-    # pod 'React-Core/Hermes', :path => '../node_modules/react-native-macos/'
-    # pod 'hermes', :path => '../node_modules/hermes-engine-darwin'
-    # pod 'libevent', :podspec => '../node_modules/react-native-macos/third-party-podspecs/libevent.podspec'
-
-    # Pods specifically for macOS target
-  end
-
-  target 'HelloWorld-iOS' do
-    platform :ios, '10'
-    use_native_modules!
-
-    # Enables Flipper.
-    #
-    # Note that if you have use_frameworks! enabled, Flipper will not work and
-    # you should disable these next few lines.
-    add_flipper_pods!
-    post_install do |installer|
-      flipper_post_install(installer)
-    end
-
-    # Pods specifically for iOS target
-  end
+  # Pods specifically for iOS target
```

Comment:
Why have this comment in a macOS file?

Reply:
Our template has always targeted both macOS and iOS. iirc @tom-un explained that this was done in order to easily test if our forked changes didn’t break any iOS code.

Reply:
Ya, I found value in testing macOS and iOS changes in various community modules.  Without the iOS target support in our template, then a developer in a dual macOS/iOS community module repo has to open two separate XCode projects to test spanning changes (PITA)